### PR TITLE
security: release notes

### PR DESCRIPTION
**Pull request by ceolin**
- Security release notes
- Information about previously published advisories.

### CODE REVIEW

**Review by stephanosio — APPROVED** [review]

**Review by stephanosio — CHANGES_REQUESTED** [review]
> Please remove blank lines at end of 'doc/security/vulnerabilities.rst'

@ceolin Can you please fix the coding guidelines check failure?

**Comment by ceolin** [comment]
@stephanosio done !

**Review by stephanosio — APPROVED** [review]

File: doc/security/vulnerabilities.rst [hunk 1]
```diff
@@ -1255,3 +1255,27 @@ This has been fixed in main for v3.2.0
 
 - `PR 48733 fix for main
   <https://github.com/zephyrproject-rtos/zephyr/pull/48733>`_
+
+CVE-2022-3806
+-------------
+
+DoS: Invalid Initialization in le_read_buffer_size_complete()
+
+- `Zephyr project bug tracker GHSA-w525-fm68-ppq3
```
Comment (d3zd3z) [inline doc/security/vulnerabilities.rst:1264]: Do we want to say "project bug tracker", or "vulnerability tracker"?

**Review by d3zd3z — APPROVED** [review]

File: doc/security/vulnerabilities.rst (hunk 1 above)
Comment (ceolin) [reply doc/security/vulnerabilities.rst:1264]: The whole document is using "project bug tracker". We can do a batch change later though.

**Pull request merged**